Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2005-09-05
Initial Package Version: 1.4.2
Upstream Status:         From upstream
Origin:                  Upstream recommendation
Description:             Fixes cURL based LDAP support

$LastChangedBy$
$Date$


diff -Naur gnupg-1.4.2-orig/g10/keyserver.c gnupg-1.4.2/g10/keyserver.c
--- gnupg-1.4.2-orig/g10/keyserver.c	2005-07-22 15:55:11.000000000 +0000
+++ gnupg-1.4.2/g10/keyserver.c	2005-09-05 05:52:21.000000000 +0000
@@ -860,7 +860,9 @@
 static const char *
 keyserver_typemap(const char *type)
 {
-  if(strcmp(type,"ldaps")==0)
+  if(strcmp(type,"ldap")==0)
+    return "ldap";
+  else if(strcmp(type,"ldaps")==0)
     return "ldap";
   else if(curl_can_handle(type))
     return "curl";
